docs/Project-Configuration/Project-Commands.md

```diff
@@ -597,9 +598,12 @@ tasks:
       - func: ssh-ready
       - func: other-tasks
 ```
-
-The mcipacker.pem key file was created by echoing the value of the
-\${\_\_project_aws_ssh_key_value} expansion into the file. This
+Note:
+- The ${admin_user_name} expansion is set to the value of the
```

Comment:
nit: enclose `${admin_user_name}` in tildes.

Reply:
Can we mention that this has to be set by the user manually, to clarify that it won't be available in the default expansions?